test/e2eERC20.js

```diff
@@ -53,8 +53,8 @@ contract('E2E ERC20 - Same Chain', async accounts => {
             Ethers.utils.hexZeroPad(Ethers.utils.hexlify(depositAmount), 32).substr(2) +
             Ethers.utils.hexZeroPad(Ethers.utils.hexlify(64), 32).substr(2) + // length of next arg in bytes
             tokenID +
-            Ethers.utils.hexZeroPad(Ethers.utils.hexlify(32), 32).substr(2) + // length of next arg in bytes
-            Ethers.utils.hexZeroPad(Ethers.utils.hexlify(recipientAddress), 32).substr(2);
+            Ethers.utils.hexZeroPad(Ethers.utils.hexlify(20), 32).substr(2) + // length of next arg in bytes
+            Ethers.utils.hexlify(recipientAddress).substr(2);
```

Comment:
Would you mind adding a comment identifying the amounts that these values are? I had a hard time earlier deciphering them.
Or pull the values in `hexlify` out as a variable